src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -284,7 +288,8 @@ private async Task<SearchResult> SearchImpl(SearchOptions searchOptions, bool hi
                                 isDeleted,
                                 null,
                                 null,
-                                null),
+                                null,
+                                searchParameterHash),
```

Comment:
super nit: it's a bit weird to see hash populated and `SearchIndices` is set to null.
Any chance we can document in future what is `SearchIndices` and `SearchParameterHash` is?

Reply:
Not a nit! I think this is really important to understand. I am currently updating the reindexing documentation to clarify this, and I'll post a link to it here once I'm done.

Reply:
A PaaS documentation PR is up [here](https://microsofthealth.visualstudio.com/Health/_git/health-paas-docs/pullrequest/14652?path=%2Fspecs%2FFHIR%2FCustomSearch%2FReindexing.md) 😊